container/dockershell.sh

```diff
@@ -57,6 +57,7 @@ OPTIONS:
                   e.g. "db2eventstore-1578174815082"
 --user         User name of the Event Store server
 --password     Password of the Event Store server
+--es-version   Event Store target cluster version 
```

Comment:
I'd add a comment specifying that this is used to pick the tag of the image to search for, given the image is built with the version as the tag.